content/developer/reference/backend/orm/changelog.rst

```diff
@@ -13,10 +19,33 @@ Odoo Online version 15.4
   and are less confusing about what is actually done in each case.
   See the section :ref:`SQL Execution <reference/orm/sql>`.
 
+Odoo Online version 15.3
+========================
+
+- Arguments name change for :meth:`~odoo.models.BaseModel.search`, :meth:`~odoo.models.BaseModel.search_count`
+  and :meth:`~odoo.models.BaseModel._search`:  `args` is called `domain` now. (`#83687 <https://github.com/odoo/odoo/pull/83687>`_)
```

Comment:
Subject first.
```suggestion
- The argument `args` is renamed to `domain` for :meth:`~odoo.models.BaseModel.search`, :meth:`~odoo.models.BaseModel.search_count`
  and :meth:`~odoo.models.BaseModel._search`. `#83687 <https://github.com/odoo/odoo/pull/83687>`_
```

Reply:
Note that I moved the PR link *after* the sentence, without parenthesis. IMO it'd be simpler and clearer like that if all PR links were systematically added at the end rather than included in the sentence.